### ISSUE TITLE
Bring selection into view in navigation mode

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#22665]
**Is your feature request related to a problem? Please describe.**
When one is in navigation mode and selects using arrows a block below the fold, the content is not scrolled into view.

**Describe the solution you'd like**
When we move selection outside of the current view (say below the fold) the content should be brought into view by scrolling.

**Describe alternatives you've considered**
None.

cc @tellthemachines

**Comment by talldan** [comment]
I think this ones a duplicate of https://github.com/WordPress/gutenberg/issues/22420
